akka-testkit-typed/src/main/scala/akka/testkit/typed/javadsl/TestProbe.scala

```diff
@@ -112,17 +92,15 @@ abstract class TestProbe[M] {
    * }
```

Comment:
code example doesn't look correct
should use a probe, for the replyTo and probe. expectMessageType

I'd rather not have code examples in scaladoc/javadoc at all